packages/manager/modules/adp/src/adp.component.js

```diff
@@ -0,0 +1,6 @@
+import template from './adp.html';
+
+export default {
+  template,
+  controllerAs: '$ctrl',
```

Comment:
In an AngularJS component, `controllerAs` is `$ctrl` by default (cf: https://docs.angularjs.org/guide/component#comparison-between-directive-definition-and-component-definition).

Reply:
This has been removed